src/main/java/fr/silverpricing/api/model/Coordinates.java

```diff
@@ -1,32 +1,59 @@
 package fr.silverpricing.api.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import lombok.*;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
```

Comment:
:information_source: No method declarations in the repository interface. (Severity: Minor)